src/main/java/ro/msg/learning/shop/service/locationselection/MostAbundantLocationSelectionAlgorithm.java

```diff
@@ -0,0 +1,34 @@
+package ro.msg.learning.shop.service.locationselection;
+
+import ro.msg.learning.shop.exception.LocationSelectionException;
+import ro.msg.learning.shop.model.OrderDetail;
+import ro.msg.learning.shop.model.Stock;
+
+import java.util.*;
```

Comment:
Replace * imports with explicit imports. We don't want to import everything from java.util. Please replace also all the * imports from the entire project with explicit import